pkgs/development/libraries/mesa/default.nix

```diff
@@ -179,20 +180,20 @@ stdenv.mkDerivation {
     mv -t $osmesa/lib/ $out/lib/libOSMesa*
   '';
 
-  # TODO:
-  #  check $out doesn't depend on llvm: builder failures are ignored
-  #  for some reason grep -qv '${llvmPackages.llvm}' -R "$out";
   postFixup = optionalString stdenv.isLinux ''
     # set the default search path for DRI drivers; used e.g. by X server
     substituteInPlace "$dev/lib/pkgconfig/dri.pc" --replace "$drivers" "${libglvnd.driverLink}"
+    substituteInPlace "$dev/lib/pkgconfig/d3d.pc" --replace "$drivers" "${libglvnd.driverLink}"
 
     # remove pkgconfig files for GL/EGL; they are provided by libGL.
     rm -f $dev/lib/pkgconfig/{gl,egl}.pc
 
-    # Update search path used by pkg-config
-    for pc in $dev/lib/pkgconfig/{d3d,dri,xatracker}.pc; do
-      if [ -f "$pc" ]; then
-        substituteInPlace "$pc" --replace $out $drivers
+    # Move pkg-config files for xatracker libraries to $driversdev
+    mkdir -p $driversdev/lib/pkgconfig
+    for pc in lib/pkgconfig/xatracker.pc; do
```

Comment:
A dedicated output (`driversdev`) for just `xatracker.pc` seems a bit strange to me. The name of the output is a bit misleading, the actual `xatracker` headers are in the `dev` output (I assume), and this part is difficult to review as we don't know which packages this might break (without running a full nixpkgs-review).